docs/ml/index.asciidoc

```diff
@@ -1,35 +1,36 @@
 [role="xpack"]
 [[xpack-ml]]
-= Machine Learning
```

Comment:
I think we should revert the change to this title.  The Kibana Guide's table of contents in intended to align with the names of the apps in Kibana, so "Machine Learning" is still the correct name. We might need to start developing separate sub-pages, however, related to the "Job management", "Data frames", "Data visualizer", "Anomaly Explorer" tabs, etc. That can be in separate PRs later though.